config/locales/fra/models.yml

```diff
@@ -460,6 +460,14 @@ fra:
           attributes: #?
             parameters: #?
               check_errored: "La connexion au partenaire a échoué. Veuillez vérifier les informations saisies ou réessayer plus tard." #?
+        intervention:
+          attributes:
+            printed_on:
+              not_opened_financial_year: "Un exercice comptable ouvert est nécessaire pour la (les) période(s) ci-dessous"
```

Comment:
That's good to fix the locale problem for this particular model. But if you put the locale in `activemodel.errors` it is used for all models in one go!